sdk/netapp/Azure.ResourceManager.NetApp/src/Custom/Models/NetAppVolumeBackupConfiguration.Serialization.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.ResourceManager.NetApp.Models
+{
+    public partial class NetAppVolumeBackupConfiguration : IUtf8JsonSerializable
+    {
+        void global::Azure.Core.IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
+        {
+            writer.WriteStartObject();
+            if (Optional.IsDefined(BackupPolicyId))
+            {
+                writer.WritePropertyName("backupPolicyId");
+                writer.WriteStringValue(BackupPolicyId);
+            }
+            if (Optional.IsDefined(IsPolicyEnforced))
+            {
+                writer.WritePropertyName("policyEnforced");
+                writer.WriteBooleanValue(IsPolicyEnforced.Value);
+            }
+            if (Optional.IsDefined(IsBackupEnabled))
+            {
+                writer.WritePropertyName("backupEnabled");
+                writer.WriteBooleanValue(IsBackupEnabled.Value);
+            }
+            writer.WriteEndObject();
```

Comment:
we need to add a segment to write vaultId in this method otherwise it will not work for older api-versions